tests/common.py

```diff
@@ -356,3 +356,18 @@ def params_with_w_coords(w_coords):
     zero_idxs = rng.choice(num_atoms, (num_atoms // 2,), replace=False)
     w_coords[zero_idxs] = 0.0
     yield params_with_w_coords(w_coords)
+
+
+def load_split_forcefields() -> Tuple[Forcefield, Forcefield, Forcefield, Forcefield]:
+    """
+    Returns the
+        OpenFF 2.0.0 ff,
+        OpenFF 2.0.0 ff with split intramolecular and intermolecular charge terms,
+        OpenFF 2.0.0 ff with all charge terms scaled by 10x,
+        OpenFF 2.0.0 ff with only intermolecular charge terms scaled by 10x.
+    """
+    ff_ref = Forcefield.load_from_file("smirnoff_2_0_0_ccc.py")
+    ff_split = Forcefield.load_from_file("tests/data/smirnoff_2_0_0_ccc_split.py")
+    ff_scaled = Forcefield.load_from_file("tests/data/smirnoff_2_0_0_ccc_scaled.py")
+    ff_inter_scaled = Forcefield.load_from_file("tests/data/smirnoff_2_0_0_ccc_inter_scaled.py")
+    return ff_ref, ff_split, ff_scaled, ff_inter_scaled
```

Comment:
nit: worth defining a dataclass for the return value? (seems like it'd be easy to get the order confused)